client/src/js/helpers/SignCheck.jsx

```diff
@@ -49,7 +49,11 @@ function SignCheck (props) {
 
   return (
     <Paper elevation={23} className={classes.root}>
-      <h2 className={classes.h2}>Can you sign for you company?</h2>
+      <h2 className={classes.h2}>Are you authorized to sign legal agreements for your company?</h2>
+      <p>
+        Typically, this means you are an executive (e.g. CEO or CTO) or officer that explicitly
```

Comment:
Typically, this means you are an executive (e.g. CEO or CTO) or officer that  **is** explicitly authorized by your company's board.